README.md

```diff
@@ -1,50 +1,56 @@
 # Music Player - Python
 
-A `Music Player` application that allows you to listen to music files stored in the computer's internal or external memory.
+A `Music Player` application that allows you to listen to musics that stored in the computer's internal or external memory..
 
-![Table Image](images/sc1.png)
+![Table Image] (images/sc1.png)
```

Comment:
This table image is not showing... Please change the image path with demo-related-assests/sc1.png